### PR TITLE
RDBS-92 bump lucene version

### PR DESCRIPTION
**Pull request by karmeli87**

### CODE REVIEW

**Comment by ayende** [comment]
Shouldn't we wait on the lucene merge until we have the other PR merged?

**Comment by karmeli87** [comment]
The lucene PRs already merged

File: src/Raven.Server/Program.cs
Comment (ppekrol) [inline src/Raven.Server/Program.cs:39]: Also set this is RachisConsensusTestBase and TestBase

**Review by ppekrol — APPROVED** [review]

File: src/Raven.Server/Program.cs
Comment (karmeli87) [reply src/Raven.Server/Program.cs:39]: done

**Comment by ppekrol** [comment]
@ayende we will be installing that on Replica, but can you review anyway?

**Pull request merged**
